packages/core/core/src/engine.ts

```diff
@@ -38,7 +41,7 @@ class Engine {
     this.isDev = this.config && this.config.blockchainConfig && (this.config.blockchainConfig.isDev || this.config.blockchainConfig.default);
 
     if (this.interceptLogs || this.interceptLogs === undefined) {
-      utils.interceptLogs(console, this.logger);
+      interceptLogs(console, this.logger);
```

Comment:
I've inlined `interceptLogs()` in this module because it's not used anywhere else within Embark.